### ISSUE TITLE
4khd (gravure, cosplayer, and models

### ISSUE THREAD

**Issue by kitsu2** [keiyoushi/tachiyomi-extensions-source#6347]
### Source name

4khd

### Source link

https://www.4khd.com/

### Source language

All

### Other details

18+/NSFW = YES

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/keiyoushi/extensions-source/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by kitsu2** [comment]
.

**Comment by Starmania** [comment]
I think this is a little bit outside of the project' scope. What do you think of this @AwkwardPeak7 ?

**Comment by 0Ha16wA5** [comment]
@Starmania Keiyoushi hosted extension for similiar sites, at least there are 26 of them.

**Comment by AwkwardPeak7** [comment]
They are technically out of scope but we already had some for such content from tachi times (and new ones added here) so shrug

**Comment by kitsu2** [comment]
So is this against the rules?

**Comment by Starmania** [comment]
No, it will only be made someone wants to.

**Comment by kitsu2** [comment]
> No, it will only be made someone wants to.

Yes I will wait for this to be added, I hope it will be added soon

**Comment by RustedMagnet** [comment]
Any Update?

**Comment by Smol-Ame** [comment]
There is no guarantee or ETA for any request. No one is assigned any work or getting paid to do anything. All the work is done by contributors volunteering their own time & efforts into fixing whatever extension they want.

**Wait patiently, migrate to a different source, learn Kotlin/Android development/webscraping and code the fix yourself, or find somewhere else that will work on demand for free for you.**
